tests/tests.yaml

```diff
@@ -0,0 +1,174 @@
+# A test is structured as follows:
+# `from` is the original list of tasks
+# `to` is the modified list of tasks
+# `new` is the expected list of tasks detected as new
+# `changes` is a list of TaskDelta<Vec<String>> that describes the expected detected
+#     changes. For each entry in `from` (after removal of common tasks), a corresponding
+#     entry in `changes` will contain an object describing the computed changes.
+# Here is an example:
+example:
+  allowed_divergence: 20
+  from:
+    - do a thing
+    - eat vegetables due:2010-01-01 rec:+1d
+    - tell all my friends about todo.txt t:2010-02-01
+
+  to:
+    - x do a thing
+    - do another thing
+    - tell all my friends all about todo.txt t:2010-02-06
+    - x eat vegetables due:2010-01-01 rec:+1d
+    - x eat vegetables due:2010-01-02 rec:+1d
+
+  new:
+    - do another thing
+
+  changes:
+    - Changed: # do a thing
+      - Finished(true)
+    - Recurred: # eat vegetables due:2010-01-01 rec:+1d
+      - # x eat vegetables due:2010-01-01 rec:+1d
+        - Finished(true)
+      - # x eat vegetables due:2010-01-02 rec:+1d
+        - RecurredStrict
+        - Finished(true)
```

Comment:
You're welcome to add an issue. This is a new feature we didn't have before so I don't see why it would be relevant to add in this PR

Reply:
Indeed, I wasn't trying to suggest this should be done right now, hence the idea of adding a `# TODO`, in the same way as you did for the other “this is currently suboptimal behaviour” tests. That said if you don't see the point in such a feature I'll just add it myself later, this way your name won't be on the blame in case someone will want to know who added that TODO :)

Reply:
Hmm… Currently the display side of things is literally only copying the `Changes` in plain English, with the only added logic of pretty-printing, so that'd go against current program structure (and impede translation should we ever want to do one).

Anyway, added an issue so that it can be tracked :)